frontend/src/containers/Collection/CollectionDetail/CollectionDetail.js

```diff
@@ -1,26 +1,81 @@
 import React, { Component } from "react";
-import {
-    Button, Tabs, Tab,
-} from "react-bootstrap";
 import { Link } from "react-router-dom";
+import { connect } from "react-redux";
 import PropTypes from "prop-types";
 
+import {
+    Button, Tabs, Tab,
+} from "react-bootstrap";
 import {
     PaperCard, Reply,
 } from "../../../components";
 
+import { collectionActions } from "../../../store/actions";
+import { collectionStatus } from "../../../constants/constants";
+
 import "./CollectionDetail.css";
 
 class CollectionDetail extends Component {
     constructor(props) {
         super(props);
         this.state = {
+            // getCollectionStatus: collectionStatus.NONE,
+            userCount: 0,
+            likeCount: 0,
+            // eslint-disable-next-line react/no-unused-state
+            paperCount: 0,
             newReplyContent: "",
-            replies: this.props.thisCollection.replies,
-            isLiked: this.props.thisCollection.isLiked,
+            isLiked: false,
+            // members: [],
+            replies: [],
+            papers: [{
+                source: "recently added",
```

Comment:
How about deleting these dummy data? Now, we have related actions and reducers!

Reply:
I delete this!